.github/workflows/ci.yaml

```diff
@@ -726,72 +773,147 @@ jobs:
         run: |
           . venv/bin/activate
           python3 -m script.translations develop --all
-      - name: Run pytest (fully)
-        if: needs.info.outputs.test_full_suite == 'true'
-        timeout-minutes: 60
-        id: pytest-full
+      - name: Run pytest
```

Comment:
Looks like we're changing the order of the full run and the partial run. Maybe keep the order to avoid a larger diff here?

Reply:
Moved the partial jobs to the end of the file. Hope the diff is better now